nbqa/save_source.py

```diff
@@ -18,7 +18,7 @@
 if TYPE_CHECKING:
     from pathlib import Path
 
-CODE_SEPARATOR = "# %%"
+CODE_SEPARATOR = "# %%\n"
```

Comment:
this doesn't make a difference in this file, but `CODE_SEPARATOR` is imported in `nbqa/replace_source.py` and is used to split the cells. We need to include the newline in the splitter else a commented-out cell magic will be split into two different cells